.editorconfig

```diff
@@ -0,0 +1,13 @@
+# EditorConfig info: https://editorconfig.org/
+root = true
+
+[Makefile*]
+indent_style = tab
+
+[*.{c,cpp,h}]
+indent_style = space
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+max_line_length = 100
```

Comment:
I don't think we should set a max line length. We can quibble about what the *guidance* should be, but I think this should be left out of .editorconfig

Reply:
Pretty sure most editors treat this as a suggestion rather than hard limit.

Reply:
It looks as though Phil added the "100 char line rule" to our coding guidelines in October without consultation. I don't know where it came from, seems arbitrary to me. Why not 80 or 120? Or take a survey of the code and see what is currently used. Personally I try to stick to 80.

Reply:
Afaik, 120 is a more common limit